src/hooks/useTotalStaked.ts

```diff
@@ -0,0 +1,51 @@
+import BN from 'bn.js'
+import { useQuery, gql } from '@apollo/client'
+import { useUsers } from 'store/cache/user/hooks'
+import getActiveStake from 'utils/activeStake'
+import { Status } from 'types'
+
+const GET_TOTAL_CLAIMABLE = gql`
```

Comment:
In the past for gql projects I've seen these strings moved to a 'queries' file - @jowlee do you know if that's still convention? might be nice to keep these organized that way